.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby:
+          - '2.5'
+          - '2.6'
+          - '2.7'
+          - '3.0'
+          - 'head'
+        gemfile:
+          - 'gemfiles/Gemfile.faraday-0.x'
+          - 'gemfiles/Gemfile.faraday-1.x'
```

Comment:
garage_client supports the both faraday 0.x and 1.x.This gem supports both faraday-0.x and 1.x, but they have not a few differences in their public interface.

ref: https://github.com/lostisland/faraday/blob/main/UPGRADING.md#faraday-10

Therefore, I think that both of these versions should be checked by CI to make sure they work.